crates/swc_ecma_minifier/src/analyzer/mod.rs

```diff
@@ -468,12 +468,12 @@ where
 
         if let Expr::Ident(i) = e {
             if cfg!(feature = "debug") {
-                tracing::debug!(
-                    "Usage: `{}``; update = {:?}, assign_lhs = {:?} ",
-                    i,
-                    self.ctx.in_update_arg,
-                    self.ctx.in_assign_lhs
-                );
+                // tracing::debug!(
```

Comment:
is this intended?

Reply:
Yes, because it was too verbose.
I couldn't find the required log because of it.